inference-engine/tests/functional/plugin/myriad/single_layer_tests/static_shape_nms.cpp

```diff
@@ -0,0 +1,116 @@
+                                      public LayerTestsUtils::LayerTestsCommon {
+public:
+    static std::string getTestCaseName(const testing::TestParamInfo<StaticShapeNMSTestParam>& obj) {
+        StaticShapeNMSParam NMSParams;
+        InferenceEngine::Precision inputPrecision;
+        std::string targetDevice;
+        std::tie(NMSParams, inputPrecision, targetDevice) = obj.param;
+
+        const auto numBatches = std::get<0>(NMSParams);
+        const auto numBoxes = std::get<1>(NMSParams);
+        const auto numClasses = std::get<2>(NMSParams);
+        const auto maxOutputBoxesPerClass = std::get<3>(NMSParams);
+        const auto iouThreshold = std::get<4>(NMSParams);
+        const auto scoreThreshold = std::get<5>(NMSParams);
+
+        std::ostringstream result;
+        result << "numBatches=" << numBatches << "_";
+        result << "numBoxes=" << numBoxes << "_";
+        result << "numClasses=" << numClasses << "_";
+        result << "maxOutputBoxesPerClass=" << maxOutputBoxesPerClass << "_";
+        result << "iouThreshold=" << iouThreshold << "_";
+        result << "scoreThreshold=" << scoreThreshold << "_";
+        result << "inPrc=" << inputPrecision.name() << "_";
+        result << "targetDevice=" << targetDevice;
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        SetRefMode(LayerTestsUtils::RefMode::INTERPRETER);
+
+        StaticShapeNMSParam NMSParams;
+        std::tie(NMSParams, inPrc, targetDevice) = this->GetParam();
+
+        const auto numBatches = std::get<0>(NMSParams);
+        const auto numBoxes = std::get<1>(NMSParams);
+        const auto numClasses = std::get<2>(NMSParams);
+        const auto maxOutputBoxesPerClass = std::get<3>(NMSParams);
+        const auto iouThreshold = std::get<4>(NMSParams);
+        const auto scoreThreshold = std::get<5>(NMSParams);
+
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inPrc);
+
+        const auto inputBoxes = std::make_shared<ngraph::opset3::Parameter>(
+                ngPrc, ngraph::Shape({static_cast<size_t>(numBatches), static_cast<size_t>(numBoxes), 4}));
+        const auto inputScores = std::make_shared<ngraph::opset3::Parameter>(
+                ngPrc, ngraph::Shape({static_cast<size_t>(numBatches), static_cast<size_t>(numClasses), static_cast<size_t>(numBoxes)}));
+        const auto maxOutputBoxesPerClassConst = std::make_shared<ngraph::opset3::Constant>(
+                ngraph::element::i64, ngraph::Shape{}, maxOutputBoxesPerClass);
+        const auto iouThresholdConst = std::make_shared<ngraph::opset3::Constant>(
+                ngraph::element::f32, ngraph::Shape{}, iouThreshold);
+        const auto scoreThresholdConst = std::make_shared<ngraph::opset3::Constant>(
+                ngraph::element::f32, ngraph::Shape{}, scoreThreshold);
+
+        const auto staticShapeNMS = std::make_shared<ngraph::vpu::op::StaticShapeNonMaxSuppression>(
```

Comment:
Is there any test on dynamic NMS transformed to SSNMS?

Reply:
I have added them, but they are currently disabled due to absence of evaluate functions for NMS - #-30919